src/test/java/co/rsk/federate/btcreleaseclient/cache/PegoutSignedCacheImplTest.java

```diff
@@ -127,4 +127,29 @@ void putIfAbsent_shouldPutInCacheOnce_whenPegoutCreationRskTxHashIsTheSame() {
 
     assertSame(pegoutCreationRskTxHashTimestamp1, pegoutCreationRskTxHashTimestamp2);
   }
+
+  @Test
+  void performCleanup_shouldRemoveOnlyExpiredPegouts_whenPerformCleanupIsTriggered() throws Exception {
+    // setup cache
+    PegoutSignedCacheImpl pegoutSignedCacheImpl = new PegoutSignedCacheImpl(DEFAULT_TTL);
+    Field field = pegoutSignedCacheImpl.getClass().getDeclaredField("cache");
+    field.setAccessible(true);
+    field.set(pegoutSignedCacheImpl, cache);
+
+    // put an expired and not expired timestamp in the cache
+    Instant currentTimestamp = Instant.now();
+    Instant notExpiredTimestamp = currentTimestamp.minusMillis(
+        Duration.ofMinutes(10).toMillis());
```

Comment:
```suggestion
        Duration.ofMinutes(DEFAULT_TTL - 10).toMillis());
```
Just an idea, if you wanna make it super generic

Reply:
I find it easier to understand the way it is now.

Reply:
What do you think about using the minus method as with the unit? I found this easier to understand and more meaningful.

```suggestion
     currentTimestamp.minus(10, ChronoUnit.MINUTES);
```

Reply:
I agree, I have updated to use the `minus` method in all tests.